module-db/Database/Field.hpp

```diff
@@ -14,22 +14,16 @@ class Field
 
     Field(const char *value)
     {
-        if (value) {
+        if (value != nullptr) {
             mValue = value;
         }
     }
-    // Field(Field const&);
-
-    // Field& operator=(Field const&);
 
     ~Field()
     {}
 
     const char *getCString() const;
-    std::string getString() const
-    {
-        return mValue;
-    }
+    std::string getString() const;
```

Comment:
I'm wondering. What is the lifetime of the returned object WRT the lifetime of the Field?
If the lifetime of the returned object can be longer then we have to copy here. But then `const char *getCString()` is incorrect because it returns pointer to cstring stored in the Field which might be destroyed. However if the lifetime of the `Field` must be longer than the variable returned from the getter then we can return `const&` here to avoid copying.

Reply:
Changed the returned value to `const&` as you suggested.